client/src/screens/CategoryScreen/CategoryScreen.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {useEffect, useState} from "react";
+import {StackScreenProps} from "@react-navigation/stack";
+import {HomeStackParamList} from "../../utils/navigation-stack";
+import {deleteItem, getItems, recordItemChosen, updateItemListOrder} from "../../services/items.service";
+import {Item, ItemWithCategory} from "../../models/item";
+import {FullActionModal} from "../../components/FullActionModal/FullActionModal";
+import {CircleIcon} from "../../components/icons/CircleIcon";
+import {useIsFocused} from "@react-navigation/native";
+import useSentenceBeginning from "../../hooks/useSentenceBeginning";
+import {DragAndDrop} from "../../components/DragAndDrop/DragAndDrop";
+import {styles} from "./styles";
+import {Text, View} from "react-native";
+import {EditRemoveSwitch} from "../../components/icons/EditRemoveSwitch";
+import {DeleteModal} from "../../components/DeleteModal/DeleteModal";
+import Toast from "react-native-toast-message";
+import {getFavorites} from "../../services/favorites.service";
+import {upsertFavoritesThunk} from "../../store/favorites/favorites.slice";
+import {useAppDispatch} from "../../store/hooks";
+
+type CategoryScreenProps = StackScreenProps<HomeStackParamList, 'Category'>;
+
+export const CategoryScreen = ({navigation, route}: CategoryScreenProps) => {
+    const dispatch = useAppDispatch();
+    const [items, setItems] = useState<Item[]>([]);
+    const [isModalVisible, setModalVisible] = useState<boolean>(false);
```

Comment:
Rename to `isFullActionModalVisible, setFullActionModalVisible`

Reply:
אני יודע שזה אני.. זה היה נכון לאותה שעה..
פשוט עכשיו הוספת DeleteModal ולכן זה מבלבל

Reply:
done